site/app/components/PaginatedTable.js

```diff
@@ -141,11 +140,12 @@ export default {
     totalEntries() {
       return this.filteredRows.length;
     },
-    auctionHouseOptions() {
-      const config = this.dataFilters.find(
-        filter => filter.id === 'auction-house'
-      );
-      return config ? config.options_generator(this.rows) : [];
+    auctionHouseConfig() {
+      return this.dataFilters.find(filter => filter.id === 'auction-house');
```

Comment:
This is harcoded for now, as an incremental step.  Eventually, this will be removed and we can simply iterate through the config.

Reply:
👍